### ISSUE TITLE
use sgdr scheduler tf impl

### ISSUE THREAD

**Issue by pfistfl** [mlr-org/mlr3keras#34]
callback_learning_rate_scheduler(tf$keras$experimental$CosineDecayRestarts(0.001, 100, t_mul = 2, m_mul = .8))

**Comment by pfistfl** [comment]
Should be fixed
